docs.yaml

```diff
@@ -37,6 +37,32 @@ rewrites:
   - http://downloads.datastax.com/php-driver/1.2.0/: http://downloads.datastax.com/php-driver/
   - http://datastax.github.io/php-driver/api/Cassandra/interface.Type/: https://docs.datastax.com/en/developer/php-driver/latest/api/Cassandra/class.Type/
   - http://datastax.github.io/php-driver/features/http://localhost:8000/features/datatypes/: https://docs.datastax.com/en/developer/php-driver/latest/features/
+  - search: 'https://(helpdocs|docs).datastax.com/(en/astra/)?(aws|gcp)/(dscloud/apollo|doc/dscloud/astra)/dscloudGettingStarted.html'
```

Comment:
@hhughes The existing redirects in this file are formatted differently than what we added to the Java driver file. Should @emeliawilkinson24 change the redirects to match this format in this file?

cc @absurdfarce